prac_08/KivyDemos-master/squaring.kv

```diff
@@ -0,0 +1,21 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        orientation: 'horizontal'
```

Comment:
Nothing wrong with this however orientation: 'horizontal' not required as this is the default orientation for new box layouts